github_archive_test.go

```diff
@@ -18,8 +19,8 @@ func TestUnarchiveRepo(t *testing.T) {
 		t.Errorf("|%v|", err)
 	}
 	defer os.RemoveAll(f)
-
-	ws, err := createWorkspaceDirectory(f, "baz", "foo.bar")
+	basename := fmt.Sprintf("%v", time.Now().UnixNano())
```

Comment:
Duplicate of https://github.com/MediaMath/grim/pull/55/files#diff-9fe6a2c7832721bba3826ce78a216ebeR185

Should be a function.